examples/vite-example/package.json

```diff
@@ -18,7 +18,6 @@
     "@types/react": "^18.0.26",
     "@types/react-dom": "^18.0.9",
     "@vitejs/plugin-react": "^3.0.0",
-    "typewind": "*",
```

Comment:
this dependency key is dup, so I removed it

Reply:
Thanks for noticing!